.github/workflows/superset-helm-release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+  release:
```

Comment:
We need a way to lint the chart in order to make sure that we're not clobbering versions. Take a look at this tool: https://github.com/helm/chart-testing

It will check the current branch against master and will yell if you made chart changes, but forgot to bump the Chart version.